documentation/kyverno-cli.md

```diff
@@ -57,7 +57,8 @@ kyverno validate /path/to/policy1.yaml /path/to/policy2.yaml /path/to/folderFull
 #### Apply
 Applies policies on resources, and supports applying multiple policies on multiple resources in a single command.
 Also supports applying the given policies to an entire cluster. The current kubectl context will be used to access the cluster.
- Will return results to stdout.
+
+Will return results to stdout. Also supports saving the mutated resource in a mentioned file or directory.
```

Comment:
Use an active tense, e.g. 

Displays mutate results to stdout, by default. Use the `-o <path>` flag to save mutated resources to a file or directory.

Reply:
updated accordingly